functions/__git.default_branch.fish

```diff
@@ -3,9 +3,9 @@ function __git.default_branch -d "Fallback to main when master branch is not fou
   if set -l default_branch (command git config --get init.defaultBranch)
     and command git show-ref -q --verify refs/heads/{$default_branch}
     echo $default_branch
-  else if command git show-ref -q --verify refs/heads/master
-    echo master
-  else
+  else if command git show-ref -q --verify refs/heads/main
```

Comment:
We'll want to update the function description to match this

Reply:
Done